### ISSUE TITLE
ty: ignore[unresolved-import] fails for multi-line import

### ISSUE THREAD

**Issue by riklopfer** [astral-sh/ty#1910]
### Summary

Expected this to work per https://docs.astral.sh/ty/suppression/#ty-suppression-comments

```python
def main():
    # ignore works
    from java.util import LinkedList, ArrayList  # ty: ignore[unresolved-import]

    # ingore works
    from java.util import (  # ty: ignore[unresolved-import]
        LinkedList,
        ArrayList,
    )

    # ingore fails
    from java.util import (
        LinkedList,
        ArrayList,
    )  # ty: ignore[unresolved-import]

```

### Version

ty 0.0.1-alpha.34 (ef3d48ac4 2025-12-12)

**Comment by MichaReiser** [comment]
Thanks.

This is another instance where it would be beneficial to have an optional `suppression_range` that can be wider than the actual diagnostic range.

**Comment by alexprengere** [comment]
I have a slightly different set of results on the latest `ty` (0.0.30):

```python
# Ignore works
from ext import a, b  # ty: ignore[unresolved-import]

# Ignore fails, we have a warning about unused directive
from ext import (  # ty: ignore[unresolved-import]
    e,
    f,
)

# Ignore fails, we also have a warning about unused directive
from ext import (
    c,
    d,
)  # ty: ignore[unresolved-import]

# Ignore works
from ext import (
    g,  # ty: ignore[unresolved-import]
    h,  # ty: ignore[unresolved-import]
)
```
